device-types/Huawei/NE40E-M2K-B.yaml

```diff
@@ -0,0 +1,62 @@
+---
+manufacturer: Huawei
+model: NE40E-M2K-B
+slug: ne40e-m2k-b
+is_full_depth: false
+u_height: 2
+comments: '[NE40E-M2K-B Documentation](https://support.huawei.com/hedex/hdx.do?docid=EDOC1100168834)'
+console-ports:
+  - name: Console
+    type: rj-45
+interfaces:
+  - name: GigabitEthernet0/0/0
+    type: 1000base-t
+  - name: GigabitEthernet0/3/4
+    type: 1000gbase-x-sfpp
```

Comment:
This line onwards `1000gbase-x-sfpp` meant to be `1000base-x-sfpp`. 1Tbps in a SFP+ form factor is not here yet.

Reply:
Thank you for the review. I hope now everything works as it should,